sdk/network/arm-network-rest/package.json

```diff
@@ -23,9 +23,7 @@
   "files": [
     "dist/",
     "README.md",
-    "LICENSE",
-    "review/*",
-    "CHANGELOG.md"
+    "LICENSE"
```

Comment:
I may remember it wrong: does management packages need review/ or CHANGELOG for some automation?

Reply:
/cc @qiaozha @MaryGao

Reply:
Updated to use the standard ARM automation for the package.json

Reply:
mgmt packages do need review/ and changelog to auto generate changelogs.